### PR TITLE
FRW-19 Support Symfony packages v6.

### PR DESCRIPTION
**Pull request by geega**
- Developer(s): @geega 

- Ticket: https://spryker.atlassian.net/browse/FRW-19

- Release Group: https://release.spryker.com/release-groups/view/4549

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Install   | minor  |                       |

-----------------------------------------

### Improvements

- Increased minimal PHP version to 8.
- Adjusted `Spryker\Zed\Install\Business\Style\SprykerStyle` for compatibility with `symfony/console:^6.0`.

### CODE REVIEW

**Review by dimitriyTsemma — APPROVED** [review]

File: src/Spryker/Zed/Install/Business/Style/SprykerStyle.php [hunk 1]
```diff
@@ -349,4 +350,20 @@ protected function log($message): void
...
+     * @return int
+     */
+    protected function strlenWithoutDecoration(OutputFormatterInterface $formatter, ?string $string): int
+    {
+        if (method_exists(Helper::class, 'strlen')) {
+            return Helper::strlen(Helper::removeDecoration($formatter, $string));
+        }
+
+        return Helper::width(Helper::removeDecoration($formatter, $string));
+
```
Comment (mstfEmkSpryker) [inline src/Spryker/Zed/Install/Business/Style/SprykerStyle.php:367]: could you please remove extra line

**Review by mstfEmkSpryker — CHANGES_REQUESTED** [review]

**Comment by geega** [comment]
CI will be fixed after merge master into this branch

File: src/Spryker/Zed/Install/Business/Style/SprykerStyle.php [hunk 2]
```diff
@@ -349,4 +350,19 @@ protected function log($message): void
...
         }
     }
+
+    /**
+     * @param \Symfony\Component\Console\Formatter\OutputFormatterInterface $formatter
+     * @param string|null $string
+     *
+     * @return int
+     */
+    protected function strlenWithoutDecoration(OutputFormatterInterface $formatter, ?string $string): int
```
Comment (asmarovydlo) [inline src/Spryker/Zed/Install/Business/Style/SprykerStyle.php:360]:
```suggestion
    protected function strlenWithoutDecoration(OutputFormatterInterface $formatter, string $string): int
```

File: composer.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
         "spryker/kernel": "^3.30.0",
         "spryker/log": "^3.0.0",
         "spryker/monolog": "^2.0.0",
-        "spryker/symfony": "^3.5.0"
+        "spryker/symfony": "^3.5.0",
+        "composer/semver": "^3.0.0"
```
Comment (asmarovydlo) [inline composer.json:12]: Why it was added as a dependency?

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
Please add release table, changelogs, tags.

File: composer.json (hunk 3 above)
Comment (geega) [reply composer.json:12]: 🤔 
removed

Comment (asmarovydlo) [reply composer.json:12]: The dependency is still there, please verify

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
1. Please add a release table and changelogs.
2. Please fix GithubActions, the error is related.

**Review by asmarovydlo — APPROVED** [review]

**Comment by asmarovydlo** [comment]
https://github.com/spryker/install/pull/44#issuecomment-1285309501  it's time to merge :) @geega

**Comment by geega** [comment]
@asmarovydlo Currently already merged. Don't have error in CI jobs. Only codecov about test coverage 
and Spryker CV 
```
PHP min version `>=7.4` is not expected `>=8.0`
```

**Comment by asmarovydlo** [comment]
[Note for the releaser] https://github.com/spryker/install/pull/44/checks?check_run_id=10140182143 fails are false-positive, they are related to untouched functionality

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -71,7 +70,7 @@ jobs:
       fail-fast: false
       matrix:
         php-version: [
-            '7.4'
+            '8.1'
```
Comment (asmarovydlo) [inline .github/workflows/ci.yml:73]: Now prefer lowest is the same, as the main job, please adjust to 8.0

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: composer.json [hunk 5]
```diff
@@ -4,15 +4,15 @@
     "description": "Install module",
     "license": "MIT",
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
         "spryker/kernel": "^3.30.0",
```
[the remark this answers is not in the document]
Comment (geega) [reply composer.json:8]: Added

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**